refactor(readme): reference awesome-ratatui instead of wiki (#689)
ratatui-org/ratatui: README.md

## Apps

> Checkout [awesome-ratatui](https://github.com/ratatui-org/awesome-ratatui) for a curated list of

Reviewer: One last comment, I don't know which one is grammatically correct :D

```suggestion
Check out [awesome-ratatui](https://github.com/ratatui-org/awesome-ratatui) for a curated list of
```

Author: You're right, typo from me.

If I'm not mistaken "check out" is basically "to look at" while "checkout" is usually going to the cashier or when leaving an hotel (I don't know why git checkout is named so)

> awesome apps built with ratatui!

Reviewer: ```suggestion
awesome apps/libraries built with ratatui!
```

> awesome apps/libraries built with ratatui!

Reviewer: nit:

```suggestion
awesome apps/libraries built with `ratatui`!
```

Author: It's inconsistent anyway, sometimes its "Ratatui" and sometimes "`ratatui`".

Still accepted to avoid adding another inconsistent variation
